test/utils/utils.hpp

```diff
@@ -3,28 +3,27 @@
 // SPDX-License-Identifier: Apache-2.0
 #pragma once
 
-#include <cstdint>
-#include <string>
+#include <evmc/hex.hpp>
 
 using bytes = std::basic_string<uint8_t>;
 using bytes_view = std::basic_string_view<uint8_t>;
 
-/// Encode a byte to a hex string.
-inline std::string hex(uint8_t b) noexcept
+inline auto hex(uint8_t b)
```

Comment:
This is noexcept upstream.

Reply:
Fixed.

Reply:
So this function is just not needed?

Reply:
This is now done with `using evmc::hex`. Sometimes I forget simple ways of doing things in C++.

Reply:
Ah, missed those lines in the diff.